### PR TITLE
new circuits for signer

### PR DESCRIPTION
**Pull request by zqhxuyuan**
---

Before we can merge this PR, please make sure that all the following items have been checked off:

- [ ] Linked to an issue with discussion and accepted design OR have an explanation in the PR that describes this work.
- [ ] Added **one** line describing your change in [`CHANGELOG.md`](https://github.com/manta-network/manta-rs/blob/main/CHANGELOG.md) and added the appropriate `changelog` label to the PR.
- [ ] Re-reviewed `Files changed` in the GitHub PR explorer.
- [ ] Checked that changes and commits conform to the standards outlined in [`CONTRIBUTING.md`](https://github.com/manta-network/manta-rs/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: manta-accounting/src/transfer/utxo/v3.rs [hunk 1]
```diff
@@ -2232,6 +2232,18 @@ where
 // }
 
 /// Identifier
+#[cfg_attr(
```
Comment (zqhxuyuan) [inline manta-accounting/src/transfer/utxo/v3.rs:2235]: Only add this serialize to v3 as signer use it. Maybe add to v2 too.

File: manta-accounting/src/transfer/mod.rs [hunk 2]
```diff
@@ -100,10 +100,10 @@ pub trait Configuration {
     type Compiler: Assert;
 
     /// Asset Id Type
-    type AssetId: Clone + Ord;
+    type AssetId: Clone + Ord + Debug;
```
Comment (SupremoUGH) [inline manta-accounting/src/transfer/mod.rs:103]: You don't need to add `Debug` here.

File: manta-accounting/src/transfer/mod.rs [hunk 3]
```diff
@@ -100,10 +100,10 @@ pub trait Configuration {
...
 
     /// Asset Value Type
-    type AssetValue: AddAssign + Clone + Default + PartialOrd + Sum;
+    type AssetValue: AddAssign + Clone + Default + PartialOrd + Sum + Debug;
```
Comment (SupremoUGH) [inline manta-accounting/src/transfer/mod.rs:106]: No need for `Debug` here either.

**Review by SupremoUGH — CHANGES_REQUESTED** [review]

File: manta-accounting/src/transfer/mod.rs (hunk 3 above)
Comment (Apokalip) [reply manta-accounting/src/transfer/mod.rs]: Doesn't the javascriptcompatability macro require it or atheist FMT

Comment (zqhxuyuan) [reply manta-accounting/src/transfer/mod.rs]: I remember previous it compiled error on signer, so add `Debug` trait. But now I remove `Debug` trait, and it compiled ok now.

**Pull request closed without merging**